.changeset/proud-rules-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
```

Comment:
```suggestion
'@primer/react': major
```

Reply:
@joshblack - I didn't realize this was a breaking change. Should I switch the branch to `next-major` and convert from a "draft" PR?